wordpress-mobile/WordPress-iOS
# Issue/fix crash on presenting options

This fix a crash on the Aztec editor, when tapping twice on the Headers Icon or Bullets icon make the app crash. This only happen on the iPad

To test:
 - Start the app on the iPad
 - Start the Aztec editor
 - Tap on the Headers option in the toolbar
 - Tap on the Headers options again
 - Check the app doesn't crash
 - Try the same but now press another icon on the toolbar
 - Check that everything works.

Needs review: @frosty
